esp-wifi/src/esp_now/mod.rs

```diff
@@ -809,6 +855,7 @@ unsafe extern "C" fn rcv_cb(
     });
 }
 
+#[instability::unstable]
 pub use asynch::SendFuture;
```

Comment:
This will result in something like #3856. Let's remove the asynch module instead.